### PR TITLE
:memo: Improves Documentation of Alpine Methods

### PR DESCRIPTION
**Pull request by ekwoka**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Review by Semigradsky — APPROVED** [review]
I don't know why I see this PR in my notifications but LGTM!

File: types/alpinejs/index.d.ts [hunk 1]
```diff
@@ -224,9 +224,32 @@ export interface ReactiveEffect<T = any> {
 }
 
 export interface Alpine {
+    /**
+     * Wraps an object in a reactive proxy
+     *
+     * @param obj an object to be wrapped
+     * @returns a reactive proxy
+     **/
```
Comment (Semigradsky) [inline types/alpinejs/index.d.ts:232]: `     */` instead

**Comment by dygy** [comment]
> I don't know why I see this PR in my notifications but LGTM!
same shit bro :DDDD

**Comment by ekwoka** [comment]
😆 

I guess just from having recent commits somehow the bot/github messed up

**Review by rbuckton — APPROVED** [review]

**Pull request closed without merging**